common/scripts/x86_64/CentOS_7/installMaster.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -e
```

Comment:
why are we moving installMaster.sh to {arch}/{os}? this is not present in https://github.com/Shippable/admiral/tree/master/common/scripts/Ubuntu_14.04 and this file doesn't have any OS specific commands